package.json

```diff
@@ -10,7 +10,13 @@
         "test": "karma start"
     },
     "dependencies": {
+        "@babel/core": "^7.0.0-beta.40",
+        "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.40",
+        "@babel/preset-env": "^7.0.0-beta.40",
         "axe-core": "^2.6.1",
+        "babel-core": "^6.26.0",
```

Comment:
Why are these here?

Reply:
I was referring to the older babel-core and babel-plugin-transform-object-rest-spread deps.